recipes/qat_distributed.py

```diff
@@ -137,6 +133,9 @@ def __init__(self, cfg: DictConfig) -> None:
         # Training cfg
         self._resume_from_checkpoint = cfg.resume_from_checkpoint
         self._gradient_accumulation_steps = cfg.gradient_accumulation_steps
+        self._fsdp_sharding_strategy = torch.distributed.fsdp.ShardingStrategy[
+            cfg.get("fsdp_sharding_strategy", "FULL_SHARD")
+        ]
```

Comment:
ah crap we don't need this (sorry I should've deleted from the full finetune recipe too). This is supplanted by the fsdp_reshard_after_forward config